cmake/modules/StandaloneCrt.cmake

```diff
@@ -28,9 +28,12 @@ if(USE_MICRO)
          "3rdparty/dlpack/include *.h -> include"
          "3rdparty/dmlc-core/include *.h -> include"
          "include/tvm/runtime c_*_api.h -> include/tvm/runtime"
+         "include/tvm/runtime metadata*.h -> include/tvm/runtime"
```

Comment:
did we wind up needing metadata_base? I think maybe could update this to just the C header now